ColorPatterns/Scenes/Game/GameViewModel.swift

```diff
@@ -14,10 +14,11 @@ protocol GameViewModelDelegate: class {
 
 final class GameViewModel {
 
+    // TODO: Make a protocol for this class and then this class implementing that protocol, so you can use it in controller
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameViewModel.swift#L17"></span>TODOs should be resolved (Make a protocol for this class…).<br />
<code>todo</code> <code>GameViewModel.swift:17</code></td>
    </tr>
  </tbody>
</table>